### PR TITLE
Remove validate_pierone_url so CLI works with other docker registries

### PR DESCRIPTION
**Pull request by jmcs**
Removes check for swagger.json to allow for registries that don't expose it.

### CODE REVIEW

File: pierone/cli.py [hunk 1]
```diff
@@ -35,17 +35,6 @@ def print_version(ctx, param, value):
     ctx.exit()
 
 
-def validate_pierone_url(url: str) -> None:
-    ping_url = url.rstrip('/') + '/swagger.json'
-    try:
-        response = requests.get(ping_url, timeout=5)
```
Comment (omartrigui) [inline pierone/cli.py:41]: I believe it should have been `requests.head` instead.

Comment (jmcs) [reply pierone/cli.py:41]: It should but we are removing it completly

**Comment by omartrigui** [comment]
:+1:

**Comment by jmcs** [comment]
:+1:

**Pull request merged**